docs/data/material/guides/interoperability/interoperability.md

```diff
@@ -689,6 +689,34 @@ export default function PlainCssPriority() {
 
 **Note:** If you are using styled-components and have `StyleSheetManager` with a custom `target`, make sure that the target is the first element in the HTML `<head>`. If you are curious to see how it can be done, you can take a look at the [`StyledEngineProvider`](https://github.com/mui/material-ui/blob/master/packages/mui-styled-engine-sc/src/StyledEngineProvider/StyledEngineProvider.js) implementation in the `@mui/styled-engine-sc` package.
 
+5. Change the target container for `Portal`-related elements to appear in the react tree properly.
+
+```jsx
+const root = ReactDOM.createRoot(document.getElementById('root'));
+const theme = createTheme({
+  components: {
+    MuiPopover: {
+      defaultProps: {
+        container: root,
+      },
+    },
+    MuiPopper: {
+      defaultProps: {
+        container: root,
+      },
+    },
+  },
+});
+
+root.render(
+  <ThemeProvider theme={theme}>
+    <App />
+  </ThemeProvider>;
+);
+```
+
+**Note:** To see more examples of how to use containers and encapsulation check out the [shadow DOM documentation](/material-ui/guides/shadow-dom/).
```

Comment:
I wouldn’t link this here. The solution of the two problems is similar, but it nay be confusing for the people not aware with the shadow dom.

Reply:
Got it, removed 👍